### DISCUSSION TITLE
Is there pdf available?

### DISCUSSION THREAD

**Discussion by pkonkol** [ryan4yin/nixos-and-flakes-book#28] [Q&A] [answered]

**Comment by ryan4yin** [comment]
@pkonkol currently, nope.

**Comment by ryan4yin** [comment, marked as answer]
@pkonkol Tried to generate an experimental pdf document, it‘s basically readable, with some flaws:

https://github.com/ryan4yin/nixos-and-flakes-book/releases

**Reply by boblehest** [reply]
Should this perhaps be added to CI/CD before marking #24 as solved? Otherwise the pdf will not be up to date.

**Reply by ryan4yin** [reply]
@boblehest Fixed by https://github.com/ryan4yin/nixos-and-flakes-book/pull/39

**Reply by boblehest** [reply]
Fantastic :) Thanks
